docs/building-ros.md

```diff
@@ -9,23 +9,9 @@ Before we can build the workspace, install the workspace c++ library dependencie
 sudo apt-get install ros-melodic-tf2-geometry-msgs python-catkin-tools ros-melodic-rqt-multiplot ros-melodic-joy ros-melodic-cv-bridge ros-melodic-image-transport libyaml-cpp-dev libcurl4-openssl-dev
 ```
 
-Then:
-
-- Build AirSim shared c++ code. From the root of this repository, run:
-```
-cd AirSim
-./setup.sh
-./build.sh
```

Comment:
it would be nice if we could add a one-liner explaining what these do on a high level